code/web/src/modules/user/StyleSurvey.js

```diff
@@ -41,4 +90,10 @@ const StyleSurvey = () => {
 
 StyleSurvey.propTypes = {};
 
-export default StyleSurvey;
+function profileState(state) {
+  return {
+    user: state.user,
+  };
+}
+
+export default connect(profileState, null)(StyleSurvey);
```

Comment:
Add  to set the redux state so that the component had access to the initial redux state - ie the use